pyqtgraph/widgets/PenSelectorDialogbox.py

```diff
@@ -0,0 +1,114 @@
+        self.doubleSpinBoxPenWidth.setValue(1)
+        self.doubleSpinBoxPenWidth.setMaximum(9999.99)
+        self.doubleSpinBoxPenWidth.setObjectName(_fromUtf8("doubleSpinBoxPenWidth"))
+        self.gridLayout.addWidget(self.doubleSpinBoxPenWidth, 0, 1, 1, 1)
+        self.comboBoxPenStyle = QtGui.QComboBox(Dialog)
+        self.comboBoxPenStyle.setObjectName(_fromUtf8("comboBoxPenStyle"))
+        self.gridLayout.addWidget(self.comboBoxPenStyle, 3, 1, 1, 1)
+        self.comboBoxPenCapStyle = QtGui.QComboBox(Dialog)
+        self.comboBoxPenCapStyle.setObjectName(_fromUtf8("comboBoxPenCapStyle"))
+        self.gridLayout.addWidget(self.comboBoxPenCapStyle, 4, 1, 1, 1)
+        self.label = QtGui.QLabel(Dialog)
+        self.label.setObjectName(_fromUtf8("label"))
+        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
+        self.label_5 = QtGui.QLabel(Dialog)
+        self.label_5.setObjectName(_fromUtf8("label_5"))
+        self.gridLayout.addWidget(self.label_5, 1, 0, 1, 1)
+        self.labelPenPreview = QtGui.QLabel(Dialog)
+        self.labelPenPreview.setMinimumSize(QtCore.QSize(160, 64))
+        self.labelPenPreview.setMaximumSize(QtCore.QSize(160, 64))
+        self.labelPenPreview.setText(_fromUtf8(""))
+        self.labelPenPreview.setObjectName(_fromUtf8("labelPenPreview"))
+        self.gridLayout.addWidget(self.labelPenPreview, 6, 0, 1, 2)
+        self.buttonBoxAcceptCancel = QtGui.QDialogButtonBox(Dialog)
+        sizePolicy = QtGui.QSizePolicy(QtGui.QSizePolicy.Fixed, QtGui.QSizePolicy.Fixed)
+        sizePolicy.setHorizontalStretch(0)
+        sizePolicy.setVerticalStretch(0)
+        sizePolicy.setHeightForWidth(self.buttonBoxAcceptCancel.sizePolicy().hasHeightForWidth())
+        self.buttonBoxAcceptCancel.setSizePolicy(sizePolicy)
+        self.buttonBoxAcceptCancel.setOrientation(QtCore.Qt.Horizontal)
+        self.buttonBoxAcceptCancel.setStandardButtons(QtGui.QDialogButtonBox.Cancel|QtGui.QDialogButtonBox.Ok)
+        self.buttonBoxAcceptCancel.setObjectName(_fromUtf8("buttonBoxAcceptCancel"))
+        self.gridLayout.addWidget(self.buttonBoxAcceptCancel, 7, 0, 1, 2)
+        self.pushButtonPenColor = ColorButton(Dialog)
+        self.pushButtonPenColor.setMinimumSize(QtCore.QSize(0, 24))
+        self.pushButtonPenColor.setMaximumSize(QtCore.QSize(16777215, 24))
+        self.pushButtonPenColor.setText(_fromUtf8(""))
+        self.pushButtonPenColor.setObjectName(_fromUtf8("pushButtonPenColor"))
+        self.gridLayout.addWidget(self.pushButtonPenColor, 1, 1, 1, 1)
+
+        self.retranslateUi(Dialog)
+        QtCore.QObject.connect(self.buttonBoxAcceptCancel, QtCore.SIGNAL(_fromUtf8("accepted()")), Dialog.accept)
+        QtCore.QObject.connect(self.buttonBoxAcceptCancel, QtCore.SIGNAL(_fromUtf8("rejected()")), Dialog.reject)
+        QtCore.QMetaObject.connectSlotsByName(Dialog)
+
+    def retranslateUi(self, Dialog):
+        Dialog.setWindowTitle(_translate("Dialog", "Dialog", None))
```

Comment:
Removing the _translate methods.
```suggestion
        Dialog.setWindowTitle("Dialog")
```